tests/baselines/reference/doubleUnderscoreSymbolsAndEmit.errors.txt

```diff
@@ -0,0 +1,60 @@
+tests/cases/compiler/index.tsx(33,1): error TS2308: Module "./b" has already exported a member named '__foo'. Consider explicitly re-exporting to resolve the ambiguity.

```

Comment:
Yep. I wanted the js output of the enum (which was bugged) and this diagnostic (which used to have an extra underscore) - I could split it into two tests if you'd prefer.